h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -855,10 +855,7 @@ private GLMModel runGLMModel(boolean standardize, Family family, Link link, Key<
   private void buildModel() {
     _model = new GLMModel(_result, _parms, this, _state._ymu, _dinfo._adaptedFrame.lastVec().sigma(), _lmax, _nobs);
     _model._output.setLambdas(_parms);  // set lambda_min and lambda_max if lambda_search is enabled
-    
-    // clone2 so that I don't change instance which is in the DKV directly
-    // (clone2 also shallow clones _output)
-    _model.clone2().delete_and_lock(_job._key);
+    _model.delete_and_lock(_job);
   }
```

Comment:
What is the problem changing the instance in the DKV directly?  Is it going to be slower?  I am just curious why this is done by Tomas.

Reply:
it does not make any sense to me, thats why I am removing it